sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -351,8 +351,8 @@ class BigQueryWrapper(object):
 
   HISTOGRAM_METRIC_LOGGER = MetricLogger()
 
-  def __init__(self, client=None, temp_dataset_id=None, temp_table_ref=None):
-    self.client = client or BigQueryWrapper._bigquery_client(PipelineOptions())
+  def __init__(self, client=None, temp_dataset_id=None, temp_table_ref=None, project_id = "apache-beam-testing"):
+    self.client = client or BigQueryWrapper._bigquery_client(PipelineOptions(project = project_id))
```

Comment:
I don't think this is needed anymore, and if it is, set default to None and pass "apache-beam-testing" from caller